File: android/2048/src/main/java/com/dreampany/play2048/ui/fragment/LicenseFragment.java
package com.dreampany.play2048.ui.fragment;

import android.os.Bundle;

import com.artitk.licensefragment.model.LicenseID;
import com.artitk.licensefragment.support.v4.RecyclerViewLicenseFragment;
import com.dreampany.framework.misc.ActivityScope;
import com.dreampany.framework.ui.fragment.BaseFragment;
import com.dreampany.framework.util.FragmentUtil;
import com.dreampany.play2048.R;

import org.jetbrains.annotations.Nullable;

import java.util.ArrayList;

import javax.inject.Inject;

/**
 * Created by Hawladar Roman on 3/8/18.
 * Dreampany Ltd
 * dreampanymail@gmail.com
 */
@ActivityScope
public class LicenseFragment extends BaseFragment {

    @Inject
    public LicenseFragment() {

    }

    @Override
    public int getLayoutId() {
        return R.layout.fragment_license;
    }

    @Override
    protected void onStartUi(@Nullable Bundle state) {
        ArrayList<Integer> licenseIds = new ArrayList<>();
        licenseIds.add(LicenseID.GSON);
        licenseIds.add(LicenseID.OKHTTP);
        licenseIds.add(LicenseID.RETROFIT);


        RecyclerViewLicenseFragment fragment = FragmentUtil.getFragment(this, R.id.fragment);
        fragment.addLicense(licenseIds);
    }

    @Override
    protected void onStopUi() {

    }
}

File: android/history/src/main/kotlin/com/dreampany/history/ui/fragment/LicenseFragment.kt
package com.dreampany.history.ui.fragment

import android.os.Bundle
import com.artitk.licensefragment.model.LicenseID
import com.artitk.licensefragment.support.v4.RecyclerViewLicenseFragment
import com.dreampany.history.R
import com.dreampany.frame.misc.ActivityScope
import com.dreampany.frame.ui.fragment.BaseFragment
import com.dreampany.frame.util.FragmentUtil
import java.util.*
import javax.inject.Inject

/**
 * Created by roman on 2019-08-03
 * Copyright (c) 2019 bjit. All rights reserved.
 * hawladar.roman@bjitgroup.com
 * Last modified $file.lastModified
 */
@ActivityScope
class LicenseFragment @Inject constructor() : BaseFragment() {

    override fun getLayoutId(): Int {
        return R.layout.fragment_license
    }

    override fun onStartUi(state: Bundle?) {
        val licenseIds = ArrayList<Int>()
        licenseIds.add(LicenseID.GSON)
        licenseIds.add(LicenseID.OKHTTP)
        licenseIds.add(LicenseID.RETROFIT)


        val fragment: RecyclerViewLicenseFragment? = FragmentUtil.getFragment(this, R.id.fragment)
        fragment?.addLicense(licenseIds)
    }

    override fun onStopUi() {

    }
}